src/main/java/com/googleinterns/zoomtube/utils/LectureUtil.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.googleinterns.zoomtube.data.Lecture;
+
+/** Utility for creating Lecture Entities and Lectures. */
+public final class LectureUtil {
+  /** Used to create a Lecture Entity and its fields. */
+  public static final String KIND = "Lecture";
+  public static final String NAME = "lectureName";
+  public static final String VIDEO_URL = "videoUrl";
+  public static final String VIDEO_ID = "video-id";
+  public static final String ID = "id";
+
+  private LectureUtil(){};
```

Comment:
```suggestion
  private LectureUtil() {};
```

Reply:
Also, please move this at the bottom as it is not important. Typically you want public methods first as this is what will be used most.

The order you choose for the members and initializers of your class can have a great effect on learnability. However, there's no single correct recipe for how to do it; different classes may order their contents in different ways.

What is important is that each class uses some logical order, which its maintainer could explain if asked. For example, new methods are not just habitually added to the end of the class, as that would yield "chronological by date added" ordering, which is not a logical ordering.

https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents

Reply:
The formatter is preventing me from adding this space

Reply:
Oh that is fine. Thanks for letting me know.